loom/common/core/allocator.h

```diff
@@ -283,6 +280,11 @@ inline void operator delete(void *p, loom_allocator_t *a)
     lmFree(a, p);
 }
 
+#define lmNew(allocator)                new(allocator, __FILE__, __LINE__)
```

Comment:
I think I just moved it so it's after the new definitions that it uses. I'm pretty sure it was because an error I got while I was compiling something, but I can double check later.